tests/test_discovergy_calls.py

```diff
@@ -1,15 +1,203 @@
 from unittest import mock
+from flask_api import status
 from tests.buzzn_test_case import BuzznTestCase
 
 
+CONSUMPTION = [{'time': 1574982000000, 'values': {'power': 0, 'power3': -27279,
+                                                  'energyOut': 0, 'power1': 0,
+                                                  'energy': 2180256872214000,
+                                                  'power2': -2437}},
+               {'time': 1574985600000, 'values': {'power': 0, 'power3': -27279,
+                                                  'energyOut': 0, 'power1': 0,
+                                                  'energy': 2180256872214000,
+                                                  'power2': -2437}}]
+EMPTY_RESPONSE = {}
+INDIVIDUAL_CONSUMPTION = b'{"1574982000000":0,"1574985600000":0}\n'
+EMPTY_RESPONSE_BYTES = b'{}\n'
+
+# byte objects cannot be line-split
+# pylint: disable=line-too-long
+GROUP_CONSUMPTION = b'{"consumed":{"1574982000000":0,"1574985600000":0},"produced":{"1574982000000":0,"1574985600000":0}}\n'
```

Comment:
Consider using python dicts instead of raw bytes.
```suggestion
GROUP_CONSUMPTION = json.dumps({'consumed':  {'1574982000000':0,'1574985600000':0},
                                                                 'produced' : {'1574982000000':0, '1574985600000':0}})
```